src/cli/subcommands/chain_cmd/prune.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019-2025 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use crate::rpc::{self, RpcMethodExt, chain::ChainPruneSnapshot};
+use clap::Subcommand;
+use std::time::Duration;
+
+#[derive(Debug, Subcommand)]
+pub enum ChainPruneCommands {
+    /// Run snapshot GC
+    Snap {
+        #[arg(long)]
+        no_wait: bool,
```

Comment:
Let's document this argument. Also, please add the command to the CLI docs generation.

Reply:
Done.